notebook-images/base/jupyter-minimal-gpu-notebook-imagestream.yaml

```diff
@@ -23,6 +23,8 @@ spec:
       kind: DockerImage
       name: quay.io/opendatahub/workbench-images:cuda-jupyter-minimal-ubi9-python-3.9-2023a-weekly
     name: "2023.1-cuda-11.8"
+    importPolicy:
+      scheduled: true
     referencePolicy:
       type: Local
```

Comment:
change referencePolicy to `type: Source `to be independent of the internal openshift registry. I think we can safely assume quay.io will always be available :-)